tests/hello_world_gles_compute.cpp

```diff
@@ -0,0 +1,823 @@
+        "    float t;"                                                                                               "\n"
+        ""                                                                                                           "\n"
+        "    // Solve dot(d, d) * t * t + 2 * dot(d, o - p) * t + dot(o - p, o - p) - r * r = 0"                     "\n"
+        "    float a = 1.0;"                                                                                         "\n"
+        "    float b = 2.0 * dot(d, o - p);"                                                                         "\n"
+        "    float c = dot(o - p, o - p) - r * r;"                                                                   "\n"
+        "    float delta = b * b - 4.0 * a * c;"                                                                     "\n"
+        "    if (delta < 0.0) {"                                                                                     "\n"
+#if 1
+        "      i++;"                                                                                                 "\n"
+#endif
+        "      continue;"                                                                                            "\n"
+        "    }"                                                                                                      "\n"
+        "    do {"                                                                                                   "\n"
+        "      t = (-b - sqrt(delta)) / (2.0 * a);"                                                                  "\n"
+        "      if (t > epsilon) break;"                                                                              "\n"
+        "      t = (-b + sqrt(delta)) / (2.0 * a);"                                                                  "\n"
+        "      if (t > epsilon) break;"                                                                              "\n"
+        "    } while (false);"                                                                                       "\n"
+        ""                                                                                                           "\n"
+        "    if (t > epsilon && t < distance) {"                                                                     "\n"
+        "      distance = t;"                                                                                        "\n"
+        "      hitPoint = ray.origin + distance * ray.direction;"                                                    "\n"
+        "      sphere = spheres[i];"                                                                                 "\n"
+        "    }"                                                                                                      "\n"
+        ""                                                                                                           "\n"
+#if 1
```

Comment:
unnecessary `#if`?

Reply:
It helps me to remember what should be removed once https://crbug.com/angleproject/2972 is resolved.

Reply:
In that case please mark it with a TODO.